src/scala/com/github/johnynek/bazel_deps/Writer.scala

```diff
@@ -207,14 +223,32 @@ object Writer {
           .map(targetFor(_).name)
 
         val (exports, runtime_deps) = model.getOptions.getTransitivity match {
-          case Transitivity.Exports => (depLabels + lab, Set.empty[Label])
-          case Transitivity.RuntimeDeps => (Set(lab), depLabels)
+          case Transitivity.Exports => (depLabels, Set.empty[Label])
+          case Transitivity.RuntimeDeps => (Set.empty[Label], depLabels)
+        }
+
+        // TODO: converge on using java_import instead of java_library once
```

Comment:
same comment, can we remove and add an issue.

Reply:
Sure, created the issue and i'm keeping the comment simply as pointer in case somebody is looking at this code.